install/common.sh

```diff
@@ -87,11 +100,17 @@ function common:check_mizar_ready {
 
 function check_all_ready {
     timeout=360
+    local cluster="kubernetes"
```

Comment:
Do you think we should have check_mizar_ready as part of arktos-up (when CNIPLUGIN==mizar)?

Reply:
I believe we would have to copy over this function to arktos-up.sh. Unfortunately we can't just call this function as arktos-up.sh deploys mizar via yaml, and doesn't have access to the mizar repo's scripts.

Reply:
yeah let's keep it here for now to  get this moving. we have to add it to arktos-up and we can do that later.